src/reactpy_django/templatetags/jinja.py

```diff
@@ -0,0 +1,66 @@
+# Copyright © 2023 Innovatie Ltd. All rights reserved.
+"""
+Jinja support.
+"""
+from django.template import RequestContext, loader
+from jinja2 import pass_context
+from jinja2.ext import Extension
+from jinja2.runtime import Context
+from reactpy_django import config
+from reactpy_django.templatetags.reactpy import component
+
+#
+# Point to our non-Django analogue.
+#
+DJT_TEMPLATE = "reactpy/component.html"
```

Comment:
This does not solve the code duplication issue.

This should be stored within `reactpy_django.templatetags.reactpy` and imported into this file.

References to `"reactpy/component.html"` within `reactpy_django.templatetags.reactpy` will need to be replaced with this global.

Reply:
Ah, I misunderstood what you wanted. Getting rid of the duplication was my point too.